### ISSUE TITLE
Spoilers aren't hidden and their reasons aren't shown at all

### ISSUE THREAD

**Issue by Mikaela** [Nheko-Reborn/nheko#1042]
### Describe the bug

When I send a spoiler with a reason from FluffyChat, the spoilered part appears as a normal message, while the spoiler reason isn't shown at all.

I would also like to be able to send spoilers (especially with the Fluffy syntax seen below)

### To Reproduce

1. Send a `/spoiler whatever` from Element or `||whatever to be as spoiler reason|spoilered text goes here||` from FluffyChat
2. See the message from Nheko

### What happened?

3. Notice that `whatever` or `spoilered text goes here` is shown as normal message while `whatever to be as spoiler reason` is not shown at all.

### Expected behavior

There is a black bar, possibly containing the spoiler explanation and on click it would become the actual message to be sent as a spoiler.

### Screenshots

N/A

### Version

0.9.3-85f0ffb6

### Operating system

Linux

### Installation method

Flatpak nightly repo or download

### Desktop Environment

i3

### Did you use profiles?

- [X] Profiles used?

**Comment by Mikaela** [comment]
This appears to be fixed, spoilers have existed for a long time now.

**Comment by deepbluev7** [comment]
Well, the reason still isn't shown, but there is another issue for that, afaik.

**Comment by Mikaela** [comment]
Yes, that seems to be https://github.com/Nheko-Reborn/nheko/issues/1231, otherwise I would reopen.
